Dockerfile

```diff
@@ -46,8 +46,7 @@ RUN ./configure \
        --prefix=$PREFIX \
        --build-type=$BUILD_TYPE \
        --log-level=INFO \
-       --no-unit-tests && \
-    make -C build all install
+    make -C build all test install
```

Comment:
Isn't this missing the `&&` from before now?

Reply:
Actually, it looks like it was a bug before. It may have done The Right Thing, but only accidentally.

Reply:
Are you sure? This is building a shell command. `./configure && make` seems like the right thing, as opposed to `./configure make`.

Reply:
Looking at the full context, I think you're right. I was thinking that we're dealing with a one-command-per-line context, but we're in `RUN`.